app/controllers/imports_controller.rb

```diff
@@ -0,0 +1,73 @@
+class ImportsController < ApplicationController
+  before_action :set_import, only: [:edit, :update, :destroy, :start]
+
+  def index
+    @imports = Import.all
+  end
+
+  def new
+    @import = Import.new
+  end
+
+  def create
+    @import = Import.new(import_params)
+    if @import.save
+      redirect_to action: :index
+    else
+      render :new
+    end
+  end
+
+  def start
+    begin
+      @import.parse_file
+    rescue CSV::MalformedCSVError => e
+      error = "Some problem appeared csv #{e}"
+    else
+      ProductFeed.create_records(@import.parse_file)
```

Comment:
@Murzic yep, you'r totally right. After testing I forgot to make changes in this place. now they'r done.